docs/handcraft.rst

```diff
@@ -0,0 +1,1940 @@
+#########################
+Handcrafting Transactions
+#########################
+
+For those who wish to assemble the transaction payload "by hand".
+
+.. contents::
+    :local:
+    :depth: 1
+
+********
+Overview
+********
+
+Submitting a transaction to a BigchainDB node consist of three main steps:
+
+1. Preparing the transaction payload.
+2. Fulfilling the prepared transaction payload.
+3. Sending the transaction payload via HTTPS.
+
+Step 1 and 2 can be performed offline on the client side. That is, they do not
+require any connection to any BigchainDB node.
+
+For convenience's sake some utilites are provided to prepare and fulfill a
+transaction via the :class:`~.bigchaindb_driver.BigchainDB` class, and via the
+:mod:`~bigchaindb_driver.offchain` module.
```

Comment:
adressed in https://github.com/bigchaindb/bigchaindb-driver/pull/176/commits/4cf6b191bddf1479bfcae98301a194fabd3929f2